src/game/Player.cpp

```diff
@@ -2447,14 +2447,12 @@ bool Player::IsGroupVisiblefor (Player* p) const
 
 bool Player::IsInSameGroupWith(Player const* p) const

```

Comment:
Thats the point:
In our core Visibility.GroupMode is not defined in World.cpp, Maybe was thrown out in error.
So currently you can change this setting to whatever you want, but it will be always default mode...